CHANGELOG.md

```diff
@@ -9,6 +9,14 @@ Unreleased
 ### Fixed
 * Fixed failed to read sessionStorage Property error when the access is denied.
 
+### Changed
```

Comment:
Take a look at this change log, i think it got messed up during merging master in.